Remove DL_RUNTIME_BUG

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #91962
* #91963
* #91961
* #91967
* __->__ #91960

This macro was made a no-op in #61903 and so we should clean up the surrounding
boiler-plate which used it.

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10

peterbell10 committed:
Remove DL_RUNTIME_BUG

This macro was made a no-op in #61903 and so we should clean up the surrounding
boiler-plate which used it.

[ghstack-poisoned]

lezcano reviewed (approved):
Manual dead code elimination, cool.

lezcano commented:
@pytorchbot merge -f "The macro is not used anywhere, it's safe to skip the CI"

peterbell10 committed:
Update on "Remove DL_RUNTIME_BUG"


This macro was made a no-op in #61903 and so we should clean up the surrounding
boiler-plate which used it.

[ghstack-poisoned]

lezcano commented:
@pytorchbot merge

lezcano commented:
ok, this is just odd, no idea what's going on here.

peterbell10 committed:
Update on "Remove DL_RUNTIME_BUG"


This macro was made a no-op in #61903 and so we should clean up the surrounding
boiler-plate which used it.

[ghstack-poisoned]

peterbell10 commented:
@pytorchbot merge -f "The macro is not used anywhere, it's safe to skip CI"
